src/dialog/windows/SDL_windowsdialog.c

```diff
@@ -516,6 +516,16 @@ static void ShowFileDialog(SDL_DialogFileCallback callback, void *userdata, SDL_
 
     filters_str = win_get_filters(filters, nfilters);
 
+    DWORD flags = 0;
+    {
```

Comment:
You don't need these braces, you can just check the flags immediately following the declaration.